[FVTR] python_pty test failing intermittently on AT >= 10.0

The python_pty test is failing sometimes in our Python test case with the following error:

0:01:09 [243/392/1] test_pty failed
test test_pty failed -- Traceback (most recent call last):
  File "/home/tcbot/bot-slave/at_opt/at-next-12.0-0-alpha/lib64/python3.6/test/test_pty.py", line 109, in test_basic
    self.assertEqual(b'For my pet fish, Eric.\n', normalize_output(s2))
AssertionError: b'For my pet fish, Eric.\n' != b'For m'

This is happening on AT 10.0, 11.0 and next, I observed this on SLES 12 and Ubuntu 16.04.

--- a/configs/11.0/packages/python/sources
+++ b/configs/11.0/packages/python/sources
@@ -49,12 +49,19 @@ atsrc_get_patches ()
 		https://github.com/python/cpython/commit/7b40cb7293cb14e5c7c8ed123efaf9acb33edae2.patch \
 		issue30714.patch 0 \
 		0f55c13d34f14cb2645e60f8c70d3926 || return ${?}
+
+	# Skip test_float_with_comma, it is failing since glibc commit ID
+	# 70a6707.
+	at_get_patch \
+		http://pkgs.fedoraproject.org/cgit/rpms/python3.git/plain/00273-skip-float-test.patch \
+		a1cb79875e6c2633222dc95910e59ab8 || return ${?}
 }
 
 atsrc_apply_patches ()
 {
 	patch -p1 < python-3.6.0-getlib64s1.patch || return ${?}
 	patch -p1 < issue30714.patch || return ${?}
+	patch -p1 < 00273-skip-float-test.patch || return ${?}
 }
 
 atsrc_package_verify_make_log ()

--- a/fvtr/python/python.exp
+++ b/fvtr/python/python.exp
@@ -69,11 +69,11 @@ if { ![file isfile "$test_script"] } {
 # These tests were commonly removed from all python versions due to the fact
 # that other tests make use of the facilities that they check, so we can catch
 # the failure later in any case. In special, test_doctest, test_doctest2,
-# test_tarball and test_venv provides ambiguous results that not always
-# represent a real success or failure.
+# test_tarball, test_venv and test_pty provides ambiguous results that not
+# always represent a real success or failure.
 set common_exclude_tests "test_doctest test_doctest2 test_asyncio \
 			  test_asyncore test_concurrent_futures	\
-			  test_buffer test_tarball test_venv"
+			  test_buffer test_tarball test_venv test_pty"
 # The following sets up the python test command so that it excludes tests
 # which should not be run using FVTR.  The exclude test list is different
 # depending on the version of python being used, although some common tests

